test/e2e/common/setup-kubestellar.sh

```diff
@@ -29,6 +29,14 @@ fi
 kflex init --create-kind $disable_chatty_status
 : Kubeflex kind cluster created.
 
+:
+: -------------------------------------------------------------------------
+: Install the post-create-hooks for ocm and kubstellar controller manager
+:
+kubectl apply -f ${SRC_DIR}/../../../config/postcreate-hooks/ocm.yaml
+kubectl apply -f ${SRC_DIR}/../../../config/postcreate-hooks/kubestellar.yaml
```

Comment:
I think this one is irrelevant, because the commands below for creating the WDS do not use a PostCreateHook

Reply:
I agree that the `kubestellar` hook is not used in today's tests (which builds the KS controller manager from source). The reason that I think this line should be in this script is that it can be useful in the future, for https://github.com/kubestellar/kubestellar/issues/1590, as a 'common setup' step.

Reply:
I am not commenting on whether this is right PR for these lines, but I wanted to point out that we will need them for tests of release images.